File: Data Structures & Algorithms/top-k-elements-in-list/submission-8.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    topKFrequent(nums, k) {
        let res = [], count = new Map();
        for(let n of nums) {
            count.has(n) ? count.set(n, count.get(n) + 1) : count.set(n, 1);
            // count[n] = (count[n] || 0) + 1;
            //  if(res[count[n]]) {
            //     if(count[n])
            //         res[count[n]]
            // }
        }
        count = [...count].sort((a,b) => b[1] - a[1]);
        count.forEach(n => {
            if(k > 0) {
                res.push(n[0]);
                k--;
            }
        })
        console.log(count,res);

        return res;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-8.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
      numDict = {}
      i=0

      while i < len(nums):
        if nums[i] in numDict: 
            return [numDict[nums[i]],i]
        else: 
            numDict[target - nums[i]] = i
            i += 1

    #   return [] 

      